pubsub/rocketmq/rocketmq.go

```diff
@@ -331,7 +331,7 @@ func (r *rocketMQ) Publish(req *pubsub.PublishRequest) error {
 	if e != nil {
 		return fmt.Errorf("rocketmq message send fail because producer failed to initialize: %v", e)
 	}
-	result, e := producer.SendSync(r.ctx, msg)
+	result, e := producer.SendSync(ctx, msg)
```

Comment:
r.ctx is now unused - please remove the property from the struct

Reply:
Done.